.wp-env.json

```
@@ -0,0 +1,14 @@
+{
+  "port": 8898,
+  "testsPort": 8899,
+  "plugins": [
+    "https://downloads.wordpress.org/plugin/rtl-tester.zip",
+    "https://downloads.wordpress.org/plugin/amp.zip",
```

```suggestion
    "https://downloads.wordpress.org/plugin/amp.zip",
    "https://downloads.wordpress.org/plugin/classic-editor.zip",
```

We need to install classic editor as well.